pkg/kubelet/kubelet.go

```diff
@@ -2748,8 +2751,12 @@ func (kl *Kubelet) canResizePod(pod *v1.Pod) (bool, *v1.Pod, v1.PodResizeStatus)
 	}
 
 	if ok, failReason, failMessage := kl.canAdmitPod(otherActivePods, podCopy); !ok {
-		// Log reason and return. Let the next sync iteration retry the resize
+		// Log reason and return. Let the next sync iteration retry the resize.
+		// If cpu manager policy is static, set resize status to infeasible.
 		klog.V(3).InfoS("Resize cannot be accommodated", "pod", podCopy.Name, "reason", failReason, "message", failMessage)
+		if failReason == ErrorInconsistentCPUAllocation {
+			return false, nil, ""
```

Comment:
Should it return `Infeasible` here as commented above?

Reply:
Thanks , I will comeback on this, I think it is not needed . Will update as soon as I do some tests

Reply:
@tallclair what do you think, based with the KEP Beta update QOS, what should I return here ?